codes/c/chapter_sorting/selection_sort.c

```diff
@@ -0,0 +1,43 @@
+/**
+ * File: selection_sort.c
+ * Created Time: 2023-05-31
+ * Author: Gonglja (glj0@outlook.com)
+ */
+
+#include "../utils/common.h"
+
+/* 交换两个元素的值 */
+void swap(int *a, int *b) {
```

Comment:
Recommend implementing the swap operation in the `selectionSort()` function. Please refer to Java codes.

Reply:
OK, updated.

Reply:
The swap function is also used in `selection_sort.cpp`, do you need to change it?

Reply:
No, swap() is a built-in function in C++ std.